packages/atomic/src/components/facets-v1/atomic-timeframe/timeframe.ts

```diff
@@ -3,12 +3,3 @@ import {RelativeDate} from '@coveo/headless';
 export interface Timeframe extends RelativeDate {
   label?: string;
 }
-
-export const defaultTimeframes: () => Timeframe[] = () => [
```

Comment:
removed the default timeframes, as they are super easy to configure and it prevents us from adding an additional "with-values" prop